src/wp-includes/class-wp-user.php

```diff
@@ -515,6 +536,11 @@ public function get_role_caps() {
 
 		$wp_roles = wp_roles();
 
+		// Edge-case: In case someone calls this method before lazy initialization, we need to initialize on demand.
```

Comment:
Nit, this is two words:

```suggestion
		// Edge case: In case someone calls this method before lazy initialization, we need to initialize on demand.
```